csi/volumeutils.py

```diff
@@ -824,6 +824,10 @@ def volume_list(voltype=None):
 def mount_volume(pvpath, mountpoint, pvtype, fstype=None):
     """Mount a Volume"""
 
+    # Create subvol dir if PV is manually created
+    if not os.path.exists(pvpath):
```

Comment:
- I couldn't think through why this is needed here.
- `mount_volume` is called after `create_(subdir|block)_volume` only and so pvpath will be created.

Reply:
I'm not sure why, but without this the folder is never created when I supply the manifest for the PV and PVC